warning: in the working copy of 'logs/myoHandWheelHoldFixed-v0_0.monitor.csv', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'logs/myoHandWheelHoldFixed-v0_1.monitor.csv', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'logs/myoHandWheelHoldFixed-v0_2.monitor.csv', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'logs/myoHandWheelHoldFixed-v0_3.monitor.csv', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'logs/myoHandWheelHoldFixed-v0_4.monitor.csv', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'logs/myoHandWheelHoldFixed-v0_5.monitor.csv', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'logs/myoHandWheelHoldFixed-v0_6.monitor.csv', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'logs/myoHandWheelHoldFixed-v0_7.monitor.csv', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'logs/myoHandWheelHoldFixed-v0_8.monitor.csv', LF will be replaced by CRLF the next time Git touches it
diff --git a/logs/myoHandWheelHoldFixed-v0_0.monitor.csv b/logs/myoHandWheelHoldFixed-v0_0.monitor.csv
index fec2164..efcd5f2 100644
--- a/logs/myoHandWheelHoldFixed-v0_0.monitor.csv
+++ b/logs/myoHandWheelHoldFixed-v0_0.monitor.csv
@@ -1,30 +1,248 @@
-#{"t_start": 1753224774.0299125, "env_id": "myoHandWheelHoldFixed-v0"}
+#{"t_start": 1753283319.8592105, "env_id": "myoHandWheelHoldFixed-v0"}
 r,l,t
-1018.209174,75,1.350015
-1020.940301,75,1.9791
-1008.803841,75,2.583881
-1015.215094,75,3.201633
-1020.49203,75,3.805361
-1011.465186,75,4.430061
-1022.100663,75,5.122949
-1016.977596,75,5.830501
-1015.786147,75,6.463489
-1005.24642,75,7.104644
-1014.64146,75,7.829228
-1022.513248,75,8.469374
-1013.148015,75,9.096324
-1015.361962,75,9.807156
-1009.632244,75,10.670025
-1015.356999,75,11.542421
-1019.020847,75,12.498425
-1024.77945,75,13.336984
-1013.179611,75,14.167206
-998.769393,75,15.000518
-1007.487081,75,15.81484
-1023.920204,75,16.678379
-1024.121073,75,17.352536
-1015.44631,75,18.047298
-1013.074355,75,18.658811
-1013.747815,75,19.293501
-1016.604967,75,19.920493
-1032.681333,75,36.874123
+657.863657,75,2.797246
+661.03879,75,4.076238
+658.220801,75,5.261926
+659.414705,75,6.512312
+661.916919,75,7.670349
+657.238053,75,8.669603
+660.709018,75,9.710373
+661.653382,75,10.518982
+662.902107,75,11.35929
+653.413734,75,12.294648
+665.189545,75,13.282246
+657.879673,75,14.320305
+668.893067,75,15.318345
+663.770263,75,16.151824
+667.900656,75,16.904903
+661.33893,75,17.628704
+658.977029,75,18.342605
+659.239421,75,19.089203
+659.289997,75,19.802174
+663.661118,75,20.580979
+661.037062,75,21.296035
+665.613285,75,22.02509
+656.019326,75,22.73576
+655.255019,75,23.436865
+666.77582,75,24.131692
+662.544609,75,24.876141
+656.520331,75,25.629796
+661.485847,75,36.643132
+663.842696,75,37.354576
+659.997203,75,38.120489
+662.336374,75,38.848711
+669.62906,75,39.572797
+659.83273,75,40.283921
+658.493736,75,41.008978
+656.547928,75,41.717512
+662.057736,75,42.432197
+653.831284,75,43.127078
+671.03339,75,43.840568
+660.245316,75,44.565331
+658.549083,75,45.271205
+657.107359,75,45.982627
+655.26776,75,46.68485
+661.41262,75,47.401218
+661.060905,75,48.109348
+654.633493,75,48.812126
+663.311338,75,49.555525
+658.41281,75,50.289412
+661.175391,75,51.007157
+660.883035,75,51.721413
+662.459478,75,52.434758
+663.209698,75,53.146761
+661.915691,75,53.863832
+655.986498,75,54.570227
+662.827821,75,55.288805
+660.1224,75,65.276396
+664.575989,75,65.991436
+658.480885,75,66.696299
+662.0118,75,67.411308
+669.897351,75,68.128311
+657.330867,75,68.846764
+642.717273,75,69.548494
+665.135158,75,70.263842
+661.102034,75,70.976376
+659.869922,75,71.672154
+667.677133,75,72.400233
+669.165774,75,73.101982
+661.644846,75,73.805346
+653.010875,75,74.518105
+661.675516,75,75.221234
+670.761837,75,75.936148
+661.766978,75,76.661172
+666.281991,75,77.359
+667.387137,75,78.054491
+659.025518,75,78.772803
+660.579729,75,79.524228
+656.849978,75,80.246923
+655.329105,75,80.968757
+658.150383,75,81.684438
+657.973987,75,82.405219
+659.341484,75,83.110444
+657.976205,75,83.794072
+662.200782,75,93.434595
+664.614434,75,94.150895
+661.174032,75,94.849967
+655.127025,75,95.543665
+653.644844,75,96.279437
+661.247318,75,96.982624
+666.693957,75,97.673637
+662.97714,75,98.386587
+657.740799,75,99.10807
+665.702975,75,99.813645
+660.86829,75,100.525132
+663.793461,75,101.222604
+646.840876,75,101.917981
+666.014499,75,102.627898
+656.84167,75,103.336013
+652.726758,75,104.066931
+653.731111,75,104.764312
+661.841689,75,105.464865
+666.090855,75,106.179734
+659.108052,75,106.867022
+659.733746,75,107.574019
+668.035948,75,108.290601
+662.2865,75,108.995863
+662.905328,75,109.708369
+659.960708,75,110.434125
+659.310314,75,111.125341
+661.257712,75,111.840617
+659.337389,75,112.544838
+658.842716,75,122.07907
+660.399743,75,122.789211
+656.236655,75,123.487175
+653.898609,75,124.189269
+642.757307,75,124.891783
+656.45044,75,125.609771
+647.851646,75,126.333752
+660.80036,75,127.029667
+657.063835,75,127.73426
+663.615235,75,128.432458
+669.078094,75,129.131763
+664.860887,75,129.839722
+653.281292,75,130.553496
+655.965557,75,131.275836
+661.809913,75,131.988851
+664.504893,75,132.703804
+670.738673,75,133.418108
+663.348,75,134.146506
+650.551444,75,134.851835
+663.271196,75,135.564537
+654.737019,75,136.29425
+658.033885,75,137.002563
+655.923951,75,137.710409
+662.00871,75,138.427386
+673.580457,75,139.436429
+621.832906,75,140.212676
+651.966892,75,140.90552
+666.481324,75,141.61747
+660.036442,75,150.612827
+666.792755,75,151.284106
+656.594133,75,151.977335
+665.867874,75,152.673279
+658.593579,75,153.356144
+656.066937,75,154.048679
+662.635942,75,154.748442
+663.97834,75,155.436933
+659.943982,75,156.114404
+667.007633,75,156.791996
+664.972943,75,157.486473
+654.369071,75,158.166956
+670.184565,75,158.852739
+661.2442,75,159.530904
+662.421292,75,160.231158
+665.648209,75,160.922508
+653.554354,75,161.604311
+667.168775,75,162.333975
+654.549984,75,163.015096
+662.041531,75,163.695365
+664.235499,75,164.386341
+666.826347,75,165.069876
+663.981203,75,165.771316
+665.738868,75,166.462965
+653.200156,75,167.138069
+670.139628,75,167.812704
+662.763976,75,168.503544
+659.128672,75,177.532049
+660.927551,75,178.227333
+665.021647,75,178.904359
+670.461264,75,179.58323
+666.708324,75,180.268874
+663.449785,75,180.958894
+664.029747,75,181.641807
+661.79314,75,182.350564
+662.654644,75,183.036653
+666.040511,75,183.731977
+663.691769,75,184.420922
+664.067241,75,185.115344
+664.361251,75,185.817856
+660.19469,75,186.510838
+669.290564,75,187.202518
+671.10671,75,187.883811
+671.835087,75,188.564455
+659.792326,75,189.264059
+662.658378,75,189.949739
+668.35932,75,190.648788
+666.984768,75,191.320658
+668.294822,75,191.99912
+666.761465,75,192.674818
+665.276476,75,193.365252
+675.49355,75,194.053565
+666.129497,75,194.740391
+667.165281,75,195.413267
+665.437833,75,205.05919
+660.991071,75,205.761419
+668.952115,75,206.449145
+659.976022,75,207.170268
+660.853476,75,207.888295
+669.830235,75,208.706482
+668.223169,75,210.139878
+669.097996,75,211.008046
+666.812687,75,211.752472
+667.982174,75,212.416091
+670.140543,75,213.114236
+671.228261,75,213.78128
+664.650737,75,214.464769
+663.914309,75,215.168381
+666.372359,75,215.846867
+679.601649,75,216.52706
+663.811899,75,217.224606
+666.930284,75,217.917656
+667.914923,75,218.652057
+662.275665,75,219.400993
+677.059941,75,220.122423
+670.560257,75,220.845024
+674.024393,75,221.565044
+661.516474,75,222.263475
+667.693349,75,222.963053
+667.373195,75,223.694769
+664.522309,75,224.422177
+667.353916,75,225.12921
+668.856346,75,235.10445
+668.726431,75,235.874876
+671.910238,75,236.745879
+671.822743,75,237.562356
+670.329177,75,238.30509
+671.544301,75,239.017357
+671.698713,75,239.725327
+665.834355,75,240.473423
+666.116881,75,241.232064
+668.320011,75,241.952107
+669.815774,75,242.657698
+672.948589,75,243.370738
+664.756058,75,244.101434
+671.01088,75,244.833544
+673.509733,75,245.564882
+671.406123,75,246.268429
+665.846065,75,246.97697
+672.537093,75,247.672188
+676.271508,75,248.368892
+675.242341,75,249.21832
+670.493423,75,250.075229
+673.357211,75,250.905253
+668.799195,75,251.630954
+671.175507,75,252.311317
+673.322891,75,252.993186
+668.071408,75,253.665332
+676.791375,75,254.344675
diff --git a/logs/myoHandWheelHoldFixed-v0_1.monitor.csv b/logs/myoHandWheelHoldFixed-v0_1.monitor.csv
index 4421cd1..960b699 100644
--- a/logs/myoHandWheelHoldFixed-v0_1.monitor.csv
+++ b/logs/myoHandWheelHoldFixed-v0_1.monitor.csv
@@ -1,30 +1,248 @@
-#{"t_start": 1753224773.9540372, "env_id": "myoHandWheelHoldFixed-v0"}
+#{"t_start": 1753283319.8472893, "env_id": "myoHandWheelHoldFixed-v0"}
 r,l,t
-1018.038952,75,1.42589
-1015.221499,75,2.054976
-1018.464875,75,2.661757
-1006.705365,75,3.278508
-1021.653701,75,3.881237
-1017.788697,75,4.507934
-1007.05453,75,5.199835
-1021.037248,75,5.906376
-1004.871929,75,6.540365
-1014.494042,75,7.182525
-1017.911855,75,7.903104
-1023.789134,75,8.543236
-1023.71766,75,9.172199
-1018.340709,75,9.885042
-1012.007245,75,10.746904
-1013.871669,75,11.620304
-1025.856771,75,12.575301
-1014.108624,75,13.412859
-1015.855303,75,14.242078
-1015.124265,75,15.075393
-1005.811843,75,15.892716
-1020.78182,75,16.755257
-1017.389266,75,17.428412
-999.80709,75,18.125176
-1024.739544,75,18.733685
-1011.331395,75,19.370376
-1020.099623,75,19.995369
-1032.681333,75,36.951008
+656.04922,75,2.809168
+662.229847,75,4.091949
+667.473453,75,5.275893
+665.848415,75,6.526249
+658.19586,75,7.684283
+658.041415,75,8.683548
+663.930207,75,9.723296
+663.863112,75,10.529894
+661.767853,75,11.366213
+661.800036,75,12.30856
+662.946163,75,13.294167
+655.146032,75,14.340226
+667.081832,75,15.329291
+651.975905,75,16.16576
+657.840766,75,16.917831
+660.389224,75,17.641627
+659.944664,75,18.354526
+655.425982,75,19.100117
+661.576547,75,19.816085
+652.31816,75,20.5929
+658.681398,75,21.30896
+650.757377,75,22.032015
+657.015233,75,22.747682
+653.006807,75,23.45079
+658.15954,75,24.143614
+661.512774,75,24.886999
+651.372971,75,25.643723
+655.624671,75,36.655053
+669.162006,75,37.367504
+650.375899,75,38.133445
+666.202872,75,38.860632
+672.567427,75,39.585719
+666.276882,75,40.295843
+663.446078,75,41.018892
+659.31952,75,41.729433
+658.917327,75,42.444118
+666.083098,75,43.1405
+652.968678,75,43.852997
+659.698453,75,44.577252
+659.603699,75,45.283126
+661.470006,75,45.993546
+658.247469,75,46.698765
+667.816179,75,47.414137
+660.329972,75,48.121269
+656.898703,75,48.824047
+663.311495,75,49.569301
+654.407734,75,50.301333
+660.66977,75,51.019078
+668.171263,75,51.733334
+658.923526,75,52.447679
+668.519499,75,53.158682
+663.771843,75,53.876755
+652.944949,75,54.583142
+662.446267,75,55.300727
+657.088355,75,65.288317
+662.80581,75,66.005373
+664.493686,75,66.706207
+652.25642,75,67.423229
+662.08177,75,68.140232
+662.610829,75,68.858685
+660.16019,75,69.561456
+667.531678,75,70.275764
+657.998353,75,70.990311
+659.099299,75,71.684075
+663.440659,75,72.415886
+665.041043,75,73.113903
+666.548132,75,73.819282
+657.860268,75,74.530026
+654.175435,75,75.233155
+666.527713,75,75.947543
+662.825868,75,76.673093
+658.908693,75,77.370921
+660.690988,75,78.066412
+660.837612,75,78.784724
+654.108201,75,79.536149
+655.078928,75,80.259902
+663.678246,75,80.980678
+660.277495,75,81.696359
+658.050964,75,82.419154
+661.482567,75,83.122365
+660.905999,75,83.805993
+658.195997,75,93.446517
+660.532769,75,94.164828
+657.566126,75,94.861888
+660.560797,75,95.555586
+670.658481,75,96.293376
+662.563724,75,96.99556
+653.636629,75,97.687568
+663.559144,75,98.398509
+652.465261,75,99.119992
+662.621637,75,99.825566
+659.951405,75,100.537053
+666.027742,75,101.233006
+655.778369,75,101.929902
+654.336204,75,102.639819
+657.713584,75,103.35003
+658.421295,75,104.081476
+649.758121,75,104.776233
+660.943795,75,105.476786
+662.001734,75,106.193665
+662.974103,75,106.880962
+655.596993,75,107.58796
+660.688463,75,108.303553
+658.102588,75,109.007784
+661.722393,75,109.72029
+664.475617,75,110.446046
+661.345135,75,111.139273
+663.875928,75,111.854549
+653.978893,75,112.556759
+664.980755,75,122.090991
+667.854773,75,122.801132
+655.46227,75,123.499097
+660.406567,75,124.20119
+643.623762,75,124.903704
+652.101438,75,125.620679
+667.576596,75,126.345673
+668.677227,75,127.041588
+663.948121,75,127.748199
+657.411158,75,128.444379
+656.517219,75,129.143684
+660.773758,75,129.853656
+661.026151,75,130.565417
+658.669383,75,131.287757
+654.184938,75,132.002784
+656.866564,75,132.717757
+657.785257,75,133.432078
+660.507567,75,134.160438
+646.493024,75,134.863756
+661.388951,75,135.576458
+656.728485,75,136.306171
+652.122086,75,137.01651
+658.46505,75,137.722331
+666.164982,75,138.441324
+673.580457,75,139.446293
+642.43138,75,140.222573
+645.163893,75,140.919484
+663.261382,75,141.631409
+655.387464,75,150.624748
+665.470568,75,151.296028
+656.112282,75,151.987746
+665.458621,75,152.6852
+663.384079,75,153.368065
+660.795076,75,154.062613
+667.309685,75,154.760363
+664.232536,75,155.448854
+666.334818,75,156.124722
+664.104331,75,156.803917
+670.902195,75,157.498394
+668.386054,75,158.178877
+661.406631,75,158.866667
+660.869976,75,159.542825
+662.139862,75,160.243079
+658.207888,75,160.935588
+666.236466,75,161.617293
+663.370242,75,162.345896
+665.31069,75,163.027018
+662.027268,75,163.705273
+654.850532,75,164.398262
+662.633622,75,165.081798
+665.536197,75,165.783764
+660.718585,75,166.474886
+655.022264,75,167.147979
+667.986822,75,167.823117
+662.649685,75,168.515465
+661.157223,75,177.54397
+671.966503,75,178.239254
+669.91856,75,178.91628
+666.619189,75,179.595151
+657.462797,75,180.278787
+661.467618,75,180.970815
+668.524355,75,181.653728
+660.777589,75,182.362485
+662.912715,75,183.048574
+659.733771,75,183.745908
+661.353259,75,184.432843
+664.646993,75,185.127265
+665.960558,75,185.829777
+669.16731,75,186.522759
+668.073185,75,187.21444
+669.654021,75,187.895733
+664.35922,75,188.57436
+670.458605,75,189.27598
+666.585136,75,189.961661
+671.298865,75,190.660709
+660.502461,75,191.334599
+668.194802,75,192.01003
+662.620055,75,192.686739
+663.845856,75,193.377174
+663.60429,75,194.065486
+671.875034,75,194.752312
+661.940643,75,195.427202
+665.182345,75,205.073121
+668.99163,75,205.77334
+665.381409,75,206.461066
+671.789161,75,207.180173
+669.109767,75,207.898199
+664.925136,75,208.718403
+672.25395,75,210.156335
+670.644256,75,211.021977
+671.828236,75,211.766408
+667.0766,75,212.428012
+667.77609,75,213.126691
+666.358901,75,213.793201
+667.030267,75,214.476691
+663.618689,75,215.180302
+663.170222,75,215.860342
+671.723929,75,216.54237
+660.444832,75,217.236527
+670.071806,75,217.931593
+665.556077,75,218.663979
+662.117959,75,219.412915
+658.794684,75,220.136357
+672.179083,75,220.858958
+665.243618,75,221.576965
+671.037769,75,222.273341
+667.99462,75,222.977402
+667.81489,75,223.708772
+677.392191,75,224.434098
+668.024952,75,225.141131
+667.291096,75,235.116371
+666.592374,75,235.888898
+670.001321,75,236.759818
+668.495667,75,237.58051
+676.21318,75,238.314995
+670.902127,75,239.029278
+671.937145,75,239.739262
+674.236348,75,240.485344
+671.261185,75,241.245998
+668.827016,75,241.964029
+669.321614,75,242.669619
+671.494491,75,243.382659
+670.143302,75,244.115384
+668.180637,75,244.845465
+672.455758,75,245.576803
+672.382877,75,246.28035
+672.97536,75,246.988891
+675.810692,75,247.684109
+669.791167,75,248.380813
+672.167327,75,249.230242
+673.275102,75,250.087151
+674.442386,75,250.917174
+673.725279,75,251.642875
+671.217274,75,252.323239
+678.063245,75,253.005107
+669.885549,75,253.67524
+670.713317,75,254.360362
diff --git a/logs/myoHandWheelHoldFixed-v0_2.monitor.csv b/logs/myoHandWheelHoldFixed-v0_2.monitor.csv
index 2af4f30..d776f94 100644
--- a/logs/myoHandWheelHoldFixed-v0_2.monitor.csv
+++ b/logs/myoHandWheelHoldFixed-v0_2.monitor.csv
@@ -1,30 +1,248 @@
-#{"t_start": 1753224774.0364635, "env_id": "myoHandWheelHoldFixed-v0"}
+#{"t_start": 1753283319.839049, "env_id": "myoHandWheelHoldFixed-v0"}
 r,l,t
-1025.72838,75,1.341465
-1019.468319,75,1.973549
-1029.754,75,2.57833
-1018.906947,75,3.19708
-1021.977203,75,3.79881
-1004.523002,75,4.42351
-1020.598773,75,5.118414
-1012.181029,75,5.82095
-1008.101821,75,6.456938
-1014.794415,75,7.101101
-1007.700471,75,7.820677
-1018.359757,75,8.462823
-1012.937935,75,9.089773
-1010.102852,75,9.800605
-1019.98727,75,10.663474
-1021.245019,75,11.540877
-1020.815508,75,12.491874
-1021.558869,75,13.330433
-1013.244045,75,14.159651
-1013.636865,75,14.992966
-1015.014765,75,15.809289
-1019.313493,75,16.67283
-1015.354633,75,17.345985
-1015.455349,75,18.040747
-1026.864529,75,18.65226
-1011.083511,75,19.28695
-1008.833618,75,19.911942
-1032.681333,75,36.867572
+659.56031,75,2.817408
+653.630956,75,4.098157
+657.159295,75,5.282087
+656.138502,75,6.53449
+663.124733,75,7.69051
+656.942124,75,8.689764
+655.221277,75,9.731536
+662.713817,75,10.538134
+660.223741,75,11.374453
+673.550357,75,12.315801
+659.201463,75,13.303399
+659.017847,75,14.34147
+655.424814,75,15.339516
+665.836183,75,16.171985
+656.696756,75,16.926071
+658.105049,75,17.649867
+663.226929,75,18.363768
+662.126237,75,19.11038
+660.651815,75,19.822336
+661.470616,75,20.60114
+660.983462,75,21.3172
+654.414696,75,22.040255
+662.360281,75,22.754917
+653.6466,75,23.45903
+665.172937,75,24.151854
+664.448384,75,24.896825
+666.770649,75,25.651963
+657.554787,75,36.663294
+651.33072,75,37.373724
+655.029021,75,38.141685
+654.860407,75,38.869879
+655.738868,75,39.594963
+656.13539,75,40.304083
+663.099976,75,41.027132
+661.785884,75,41.736674
+656.512156,75,42.452358
+664.806065,75,43.148741
+663.151884,75,43.86073
+660.684772,75,44.584492
+661.894208,75,45.291366
+664.123798,75,46.002788
+659.341718,75,46.707006
+656.964236,75,47.422378
+659.838724,75,48.127511
+665.005353,75,48.83128
+654.312971,75,49.577031
+658.108824,75,50.309573
+655.771981,75,51.029313
+663.96323,75,51.740577
+654.345578,75,52.453912
+657.733567,75,53.164924
+663.862316,75,53.883994
+660.060303,75,54.591383
+657.854695,75,55.308967
+652.573975,75,65.296557
+665.346577,75,66.013613
+664.370449,75,66.71646
+667.006254,75,67.42895
+662.424348,75,68.148472
+669.249361,75,68.866925
+662.927113,75,69.569696
+651.660513,75,70.286021
+661.010736,75,70.996537
+666.354375,75,71.692316
+663.487769,75,72.420395
+663.786021,75,73.122143
+649.836755,75,73.825507
+665.930312,75,74.538266
+657.757253,75,75.241395
+654.4894,75,75.95526
+653.598936,75,76.681333
+659.491505,75,77.379162
+662.399999,75,78.074652
+653.261522,75,78.794982
+666.046547,75,79.544389
+656.826822,75,80.268143
+664.667699,75,80.992967
+653.576882,75,81.704599
+661.20063,75,82.42538
+654.956047,75,83.130606
+654.285345,75,83.814233
+652.376073,75,93.454757
+672.640394,75,94.171056
+669.126674,75,94.86812
+661.030776,75,95.563826
+657.711917,75,96.299598
+663.406946,75,97.003801
+660.349581,75,97.696828
+650.530903,75,98.406749
+647.642371,75,99.128232
+659.198543,75,99.831793
+656.35381,75,100.545293
+659.527538,75,101.242765
+662.269248,75,101.938143
+663.489511,75,102.648059
+669.811064,75,103.35721
+661.765826,75,104.087092
+657.624566,75,104.786485
+659.956626,75,105.485027
+659.18792,75,106.201905
+650.545459,75,106.887183
+655.010927,75,107.5962
+666.831364,75,108.310762
+662.992596,75,109.014014
+671.221748,75,109.728531
+661.309078,75,110.454286
+663.959516,75,111.147514
+668.844984,75,111.862789
+661.609538,75,112.562982
+662.577927,75,122.099231
+668.952942,75,122.809372
+668.134999,75,123.509346
+659.235821,75,124.20943
+666.277324,75,124.914219
+659.343624,75,125.629932
+659.718282,75,126.355514
+665.781308,75,127.049829
+664.521104,75,127.754422
+670.722027,75,128.452619
+658.66789,75,129.151925
+667.593402,75,129.861896
+664.226612,75,130.575682
+661.038125,75,131.295997
+668.499872,75,132.009012
+664.306431,75,132.723966
+659.136508,75,133.438269
+664.897559,75,134.166667
+660.716395,75,134.871997
+655.641284,75,135.584698
+658.99467,75,136.314412
+662.075824,75,137.024751
+661.160089,75,137.730571
+651.274592,75,138.447547
+673.580457,75,139.454534
+656.987435,75,140.230813
+661.58086,75,140.925682
+668.920488,75,141.639649
+658.445348,75,150.632988
+663.689091,75,151.304268
+661.233145,75,151.997497
+659.592872,75,152.69143
+660.221066,75,153.376305
+667.594429,75,154.068841
+665.268784,75,154.768603
+665.354084,75,155.457631
+659.541397,75,156.134024
+661.282944,75,156.812158
+666.90386,75,157.504615
+659.968075,75,158.189135
+660.3234,75,158.8729
+653.343714,75,159.551065
+656.828526,75,160.253336
+665.418524,75,160.940654
+660.640281,75,161.623945
+667.906686,75,162.354136
+660.231844,75,163.035258
+657.367999,75,163.715526
+659.497683,75,164.406502
+661.093546,75,165.090038
+660.621725,75,165.791477
+664.560299,75,166.483126
+667.148944,75,167.15823
+666.940446,75,167.832865
+663.102601,75,168.523705
+666.672096,75,177.552211
+657.838862,75,178.247494
+659.704365,75,178.92452
+664.461662,75,179.603391
+660.112064,75,180.289036
+661.294729,75,180.979055
+665.821121,75,181.661969
+668.068746,75,182.370725
+659.226178,75,183.056815
+662.400817,75,183.754148
+664.050579,75,184.442096
+668.477985,75,185.135506
+665.114678,75,185.838017
+666.166379,75,186.528978
+664.852387,75,187.22268
+666.474241,75,187.903973
+663.228378,75,188.584616
+661.187853,75,189.28422
+666.642029,75,189.969901
+662.846024,75,190.668949
+664.651218,75,191.340819
+663.121494,75,192.01827
+665.548357,75,192.694979
+661.334177,75,193.385414
+669.588681,75,194.075744
+664.927328,75,194.760552
+663.708382,75,195.435442
+661.360852,75,205.087396
+668.059255,75,205.779827
+665.899833,75,206.469306
+663.748231,75,207.190429
+668.608623,75,207.906439
+665.452519,75,208.728657
+672.448828,75,210.164576
+668.437959,75,211.030217
+667.875535,75,211.774649
+665.246323,75,212.436252
+663.524147,75,213.135456
+667.93459,75,213.801442
+668.75265,75,214.484931
+667.165127,75,215.188542
+673.541695,75,215.868582
+665.866035,75,216.550611
+668.896293,75,217.244767
+664.521739,75,217.939833
+667.275141,75,218.672219
+667.278325,75,219.423169
+661.421025,75,220.142584
+667.138784,75,220.867198
+663.482806,75,221.588237
+671.161397,75,222.283637
+666.829665,75,222.982168
+664.253037,75,223.719031
+662.65105,75,224.440121
+660.336507,75,225.151384
+667.472827,75,235.124612
+668.225233,75,235.895549
+668.915325,75,236.76982
+672.146234,75,237.583529
+667.915964,75,238.325252
+670.931942,75,239.037518
+671.135765,75,239.747503
+670.617714,75,240.493584
+669.562036,75,241.252226
+666.366469,75,241.972269
+670.336911,75,242.67584
+674.867886,75,243.392935
+672.185306,75,244.123624
+670.663453,75,244.85169
+673.394248,75,245.585043
+673.286316,75,246.288591
+673.413607,75,246.997131
+675.380938,75,247.692349
+676.204027,75,248.389054
+671.190402,75,249.240507
+663.631157,75,250.095391
+673.054996,75,250.9234
+667.809027,75,251.651116
+671.096848,75,252.331479
+670.228796,75,253.013347
+668.953983,75,253.685493
+674.490442,75,254.366586
diff --git a/logs/myoHandWheelHoldFixed-v0_3.monitor.csv b/logs/myoHandWheelHoldFixed-v0_3.monitor.csv
index d5e058b..0c65bfd 100644
--- a/logs/myoHandWheelHoldFixed-v0_3.monitor.csv
+++ b/logs/myoHandWheelHoldFixed-v0_3.monitor.csv
@@ -1,30 +1,248 @@
-#{"t_start": 1753224774.0151355, "env_id": "myoHandWheelHoldFixed-v0"}
+#{"t_start": 1753283319.8586583, "env_id": "myoHandWheelHoldFixed-v0"}
 r,l,t
-1016.470861,75,1.364792
-1013.88181,75,1.993877
-1009.570288,75,2.600659
-1019.092492,75,3.21741
-1014.640687,75,3.820138
-1009.599595,75,4.446836
-1027.010956,75,5.140737
-1022.783935,75,5.844277
-1006.599214,75,6.479266
-1016.256974,75,7.126419
-1005.70785,75,7.845002
-1021.213129,75,8.484151
-1023.300564,75,9.112103
-1020.069043,75,9.821933
-1020.302725,75,10.686806
-1021.044568,75,11.562205
-1007.392285,75,12.514203
-1009.999227,75,13.351761
-1016.918892,75,14.183985
-1013.034747,75,15.015295
-1023.854995,75,15.831617
-1026.001277,75,16.694158
-1016.213895,75,17.368312
-1013.735831,75,18.062075
-1018.151553,75,18.674591
-1023.678184,75,19.308278
-1013.509442,75,19.93327
-1032.681333,75,36.88991
+647.380179,75,2.795781
+652.329713,75,4.07679
+666.811854,75,5.260447
+653.440146,75,6.51488
+657.609867,75,7.672914
+661.787582,75,8.672179
+656.571651,75,9.714926
+647.388466,75,10.52053
+653.692499,75,11.354844
+666.251736,75,12.296192
+652.838843,75,13.282798
+662.726923,75,14.322867
+661.357449,75,15.318897
+659.851954,75,16.154391
+666.187403,75,16.905455
+666.065888,75,17.629256
+659.550038,75,18.343157
+662.006501,75,19.088748
+657.271416,75,19.801709
+652.372258,75,20.582542
+657.194369,75,21.297591
+662.47084,75,22.020646
+662.871944,75,22.735308
+656.598482,75,23.438434
+659.06443,75,24.132245
+650.619302,75,24.878842
+658.88928,75,25.631348
+668.013376,75,36.643684
+652.803587,75,37.357128
+656.499678,75,38.123627
+655.756294,75,38.85027
+667.580732,75,39.57435
+663.03278,75,40.285479
+655.548562,75,41.008522
+654.150464,75,41.717065
+659.10229,75,42.432749
+657.369432,75,43.129131
+645.445293,75,43.84112
+660.621061,75,44.564883
+662.870069,75,45.271757
+654.9761,75,45.981177
+669.385299,75,46.686408
+662.677191,75,47.40177
+654.314132,75,48.1099
+662.098354,75,48.813668
+653.56794,75,49.556077
+650.80065,75,50.289964
+662.931344,75,51.007709
+667.783407,75,51.721965
+654.490409,75,52.43531
+661.314305,75,53.145315
+654.025078,75,53.865386
+658.093849,75,54.571773
+661.484161,75,55.289358
+661.895576,75,65.276948
+655.330205,75,65.994004
+665.884228,75,66.698869
+656.875657,75,67.41186
+651.671424,75,68.128863
+660.796438,75,68.847316
+661.241694,75,69.549046
+659.298565,75,70.264395
+663.995104,75,70.976928
+655.404744,75,71.672706
+659.623857,75,72.400785
+660.004425,75,73.10051
+653.078633,75,73.805898
+663.606021,75,74.51664
+669.787922,75,75.219268
+667.781272,75,75.936174
+659.035149,75,76.661724
+660.349744,75,77.361997
+660.831686,75,78.055043
+653.647238,75,78.773355
+657.750798,75,79.52478
+661.668121,75,80.248533
+660.535027,75,80.971324
+669.13542,75,81.68499
+659.190828,75,82.405771
+659.19624,75,83.110477
+656.420534,75,83.794624
+653.07246,75,93.433125
+655.768398,75,94.153459
+669.841005,75,94.848511
+660.946414,75,95.543169
+655.495513,75,96.279989
+652.54752,75,96.981163
+654.073586,75,97.676199
+662.468462,75,98.38714
+653.21831,75,99.108623
+655.269935,75,99.814197
+659.582782,75,100.525684
+656.41168,75,101.223156
+660.158283,75,101.918533
+662.616731,75,102.62845
+654.114628,75,103.337601
+659.142629,75,104.069499
+657.745243,75,104.764864
+658.402234,75,105.465417
+613.595059,75,106.182296
+663.303931,75,106.869593
+661.557939,75,107.576591
+663.092623,75,108.292184
+659.238124,75,108.996415
+653.624743,75,109.708921
+657.797653,75,110.434677
+662.711555,75,111.127904
+659.779496,75,111.84318
+652.652835,75,112.547405
+657.36646,75,122.077604
+666.306894,75,122.789763
+653.911186,75,123.489737
+665.235091,75,124.189821
+667.578784,75,124.892335
+663.165709,75,125.610323
+662.323901,75,126.335905
+655.372115,75,127.030219
+660.343345,75,127.734812
+649.799449,75,128.435046
+661.055945,75,129.132315
+660.182822,75,129.840274
+655.82747,75,130.556073
+657.130105,75,131.276388
+654.628869,75,131.989403
+660.756309,75,132.704357
+659.633804,75,133.420709
+659.500343,75,134.149069
+653.490664,75,134.852387
+659.727065,75,135.565089
+652.504227,75,136.294802
+661.371828,75,137.003115
+663.986986,75,137.710962
+649.517807,75,138.427938
+673.580457,75,139.434924
+620.215211,75,140.211204
+651.913104,75,140.906072
+667.164614,75,141.618022
+662.339727,75,150.613379
+664.856059,75,151.286673
+660.247468,75,151.976377
+664.666524,75,152.673831
+671.774454,75,153.356696
+664.598527,75,154.051244
+656.076876,75,154.748994
+662.708304,75,155.435899
+673.771235,75,156.113353
+657.110838,75,156.792548
+658.968487,75,157.485006
+666.837566,75,158.169525
+661.794078,75,158.855298
+665.396218,75,159.530432
+661.899083,75,160.23171
+665.881956,75,160.924219
+664.158649,75,161.605924
+656.388996,75,162.334527
+662.26086,75,163.015649
+665.484567,75,163.695917
+656.902962,75,164.386893
+666.66435,75,165.070429
+655.155629,75,165.771868
+663.111195,75,166.465563
+667.338177,75,167.138621
+667.689161,75,167.811748
+661.918065,75,168.504096
+662.381825,75,177.530593
+663.156226,75,178.227885
+666.776057,75,178.902894
+668.457868,75,179.583782
+663.819167,75,180.269427
+671.609358,75,180.957436
+658.809576,75,181.642359
+665.282191,75,182.351116
+668.014721,75,183.037205
+672.759599,75,183.732529
+666.752804,75,184.421474
+662.755893,75,185.115896
+661.509536,75,185.818408
+661.905137,75,186.51139
+663.93224,75,187.203071
+669.22578,75,187.884364
+657.949714,75,188.562991
+665.617715,75,189.264611
+660.686485,75,189.950292
+657.936408,75,190.64934
+668.99373,75,191.32121
+668.376,75,191.999673
+665.135441,75,192.67537
+663.811099,75,193.365805
+672.768727,75,194.054117
+663.707049,75,194.740943
+663.443603,75,195.413819
+662.865493,75,205.061752
+670.562796,75,205.761971
+664.870918,75,206.449697
+672.41766,75,207.17082
+662.650118,75,207.88683
+670.291691,75,208.707034
+669.196712,75,210.14043
+665.614854,75,211.010608
+671.483013,75,211.755039
+663.786188,75,212.416643
+657.627854,75,213.116372
+665.00114,75,213.781832
+664.787552,75,214.466331
+668.644033,75,215.168933
+667.046171,75,215.848973
+668.36839,75,216.531001
+670.107876,75,217.225158
+661.244809,75,217.920224
+665.414054,75,218.654198
+667.622712,75,219.40356
+668.444967,75,220.122975
+667.010426,75,220.847589
+664.097682,75,221.56761
+666.607048,75,222.261972
+668.582912,75,222.963605
+666.993184,75,223.699422
+668.849945,75,224.420512
+666.902761,75,225.131775
+668.589868,75,235.111066
+667.368514,75,235.875089
+675.42458,75,236.748449
+673.702399,75,237.56392
+669.837431,75,238.305643
+670.404354,75,239.0159
+674.491181,75,239.728904
+671.485686,75,240.473975
+665.008187,75,241.232617
+672.153063,75,241.95266
+674.616769,75,242.65825
+669.297006,75,243.37129
+673.66572,75,244.101986
+667.739086,75,244.834096
+666.506845,75,245.565434
+671.553119,75,246.268981
+670.387219,75,246.977522
+672.959593,75,247.67274
+669.446617,75,248.367414
+670.564918,75,249.218873
+674.514233,75,250.075782
+665.674733,75,250.905805
+664.141044,75,251.632028
+667.572864,75,252.31187
+671.729649,75,252.993738
+673.872426,75,253.663871
+671.045441,75,254.346977
diff --git a/logs/myoHandWheelHoldFixed-v0_4.monitor.csv b/logs/myoHandWheelHoldFixed-v0_4.monitor.csv
index 4624d63..57a3a90 100644
--- a/logs/myoHandWheelHoldFixed-v0_4.monitor.csv
+++ b/logs/myoHandWheelHoldFixed-v0_4.monitor.csv
@@ -1,30 +1,248 @@
-#{"t_start": 1753224773.9626968, "env_id": "myoHandWheelHoldFixed-v0"}
+#{"t_start": 1753283319.839049, "env_id": "myoHandWheelHoldFixed-v0"}
 r,l,t
-1025.027865,75,1.416231
-1007.256683,75,2.047316
-1021.505493,75,2.652097
-1015.544931,75,3.269849
-1021.520222,75,3.872577
-1020.774244,75,4.499275
-1017.610995,75,5.191175
-1016.166897,75,5.896715
-1017.04601,75,6.530704
-1010.781994,75,7.172866
-1021.795854,75,7.894444
-1017.652139,75,8.53659
-1021.952054,75,9.164541
-1008.771898,75,9.876383
-1010.568484,75,10.735241
-1015.405277,75,11.613652
-1015.257161,75,12.566642
-1009.432448,75,13.405201
-1021.427443,75,14.23542
-1024.045203,75,15.067734
-1014.62946,75,15.884056
-1019.447596,75,16.746597
-1016.557287,75,17.420751
-1021.865125,75,18.115515
-1014.994007,75,18.724026
-1024.000069,75,19.360716
-1012.328282,75,19.985709
-1032.681333,75,36.941339
+643.658604,75,2.81539
+660.49019,75,4.098157
+654.033131,75,5.288168
+656.988444,75,6.53449
+655.534139,75,7.69051
+664.190371,75,8.689764
+657.233116,75,9.732544
+662.545682,75,10.537127
+665.509148,75,11.375479
+658.323958,75,12.3168
+649.671459,75,13.311396
+653.262481,75,14.344474
+655.267452,75,15.338506
+661.363268,75,16.172998
+655.264447,75,16.926071
+659.897323,75,17.649867
+661.860257,75,18.362766
+658.461361,75,19.108357
+663.651484,75,19.823323
+656.829858,75,20.60114
+663.089209,75,21.3172
+656.07802,75,22.039256
+665.484313,75,22.755922
+652.244407,75,23.457027
+657.929412,75,24.150847
+647.789549,75,24.895756
+668.025263,75,25.649957
+658.700019,75,36.663294
+662.717638,75,37.373724
+659.734078,75,38.141685
+659.06816,75,38.869879
+665.068911,75,39.592958
+658.741939,75,40.304083
+664.500625,75,41.027132
+663.77436,75,41.737673
+664.505455,75,42.453362
+662.928296,75,43.150765
+667.405301,75,43.85973
+653.588226,75,44.586492
+661.011358,75,45.294588
+664.275682,75,46.002788
+659.486504,75,46.705012
+651.580328,75,47.421379
+660.838895,75,48.127511
+662.103456,75,48.832288
+664.093528,75,49.574692
+657.353693,75,50.309573
+664.578409,75,51.028318
+663.677551,75,51.741575
+663.344294,75,52.453912
+661.381633,75,53.165926
+644.448444,75,53.885995
+662.201839,75,54.588366
+653.848383,75,55.308967
+664.535142,75,65.296557
+662.10226,75,66.011598
+650.33667,75,66.718478
+669.986445,75,67.431469
+659.876355,75,68.146279
+666.086968,75,68.866925
+663.07302,75,69.569696
+654.569575,75,70.284004
+657.769166,75,70.998551
+667.198512,75,71.692316
+658.339844,75,72.422913
+671.810084,75,73.126186
+655.872632,75,73.827523
+658.777734,75,74.538266
+644.222664,75,75.24342
+661.055912,75,75.955784
+661.80544,75,76.681333
+665.377211,75,77.379162
+663.533878,75,78.07668
+655.505769,75,78.792964
+667.642954,75,79.544389
+664.706859,75,80.26918
+653.093944,75,80.988918
+669.590872,75,81.704599
+654.379347,75,82.423362
+656.239942,75,83.131646
+663.026443,75,83.814233
+655.965539,75,93.452734
+665.712023,75,94.171056
+652.10979,75,94.870128
+664.073247,75,95.562778
+654.97134,75,96.299598
+668.114061,75,97.003801
+662.437985,75,97.696828
+653.336688,75,98.406749
+664.384265,75,99.128232
+661.484881,75,99.833806
+648.864929,75,100.545293
+668.671671,75,101.241246
+650.878062,75,101.936388
+661.386872,75,102.648059
+660.869406,75,103.35721
+658.640275,75,104.087092
+658.339567,75,104.784473
+659.144621,75,105.485027
+663.297035,75,106.201905
+653.524585,75,106.887183
+660.546576,75,107.594181
+665.418547,75,108.310762
+655.623668,75,109.016025
+662.449174,75,109.726779
+661.015926,75,110.454286
+665.291534,75,111.147514
+659.998335,75,111.860779
+668.726667,75,112.562982
+663.304146,75,122.097213
+665.671865,75,122.809372
+664.803178,75,123.509346
+664.057393,75,124.211451
+659.775651,75,124.914219
+662.180249,75,125.628919
+661.197049,75,126.353913
+672.543343,75,127.047807
+661.160782,75,127.754422
+662.436948,75,128.452619
+652.368283,75,129.151925
+657.076097,75,129.861896
+658.454375,75,130.573658
+659.088687,75,131.293981
+656.58508,75,132.009012
+660.7028,75,132.723966
+657.353372,75,133.440318
+662.903148,75,134.166667
+659.240181,75,134.871997
+660.79886,75,135.586736
+663.07116,75,136.314412
+654.038072,75,137.022724
+672.467001,75,137.730571
+661.984404,75,138.449564
+673.580457,75,139.452506
+662.547031,75,140.230813
+656.349199,75,140.925682
+656.071323,75,141.635877
+669.048962,75,150.63098
+663.59652,75,151.304268
+660.892444,75,151.993977
+672.477753,75,152.69143
+670.673737,75,153.376305
+663.75185,75,154.070853
+665.851135,75,154.770619
+670.167473,75,155.45869
+666.546379,75,156.134024
+663.084634,75,156.81164
+665.158947,75,157.504615
+671.636632,75,158.187117
+669.634626,75,158.8729
+664.765361,75,159.551065
+661.010979,75,160.251319
+664.003239,75,160.940654
+656.928606,75,161.623945
+662.318261,75,162.354136
+666.756774,75,163.035258
+666.551656,75,163.713514
+661.901135,75,164.406502
+667.864658,75,165.088026
+666.195124,75,165.792004
+663.949536,75,166.483126
+663.743074,75,167.15823
+665.546174,75,167.831357
+656.934058,75,168.523705
+667.275251,75,177.552211
+665.091036,75,178.247494
+655.546306,75,178.92452
+662.849059,75,179.60138
+668.144571,75,180.289036
+667.317167,75,180.977045
+661.093617,75,181.66398
+661.198618,75,182.370725
+670.065285,75,183.056815
+664.425322,75,183.754148
+663.63459,75,184.441083
+662.347565,75,185.135506
+664.920832,75,185.838017
+670.370082,75,186.530999
+672.112792,75,187.22268
+660.438686,75,187.903973
+663.251918,75,188.584616
+672.280434,75,189.28422
+666.515414,75,189.969901
+668.170355,75,190.666143
+668.009769,75,191.340819
+668.876994,75,192.01827
+659.166746,75,192.694979
+664.753314,75,193.385414
+660.601474,75,194.073727
+668.923249,75,194.760552
+669.475903,75,195.433428
+663.823597,75,205.081362
+668.300593,75,205.78158
+660.871728,75,206.469306
+660.020236,75,207.190429
+673.844245,75,207.906439
+659.45576,75,208.734194
+663.719601,75,210.162072
+670.220076,75,211.030217
+664.945363,75,211.774649
+669.008883,75,212.438513
+668.166494,75,213.137589
+656.239838,75,213.801442
+668.808552,75,214.484931
+671.826305,75,215.188542
+668.833958,75,215.868582
+665.606914,75,216.550611
+670.929035,75,217.244767
+672.291767,75,217.939833
+669.604438,75,218.673282
+668.871545,75,219.423169
+672.842919,75,220.144598
+663.486923,75,220.869217
+665.814128,75,221.587219
+665.182028,75,222.281581
+668.964369,75,222.983215
+656.823433,75,223.719031
+662.3247,75,224.442338
+666.865622,75,225.151384
+666.482841,75,235.12663
+664.999628,75,235.895549
+674.130693,75,236.768058
+672.528991,75,237.583529
+665.730631,75,238.325252
+675.00043,75,239.037518
+663.454597,75,239.748513
+673.409632,75,240.4956
+663.633569,75,241.254238
+671.191529,75,241.972269
+673.334243,75,242.67786
+666.483817,75,243.3909
+669.638674,75,244.121596
+668.595124,75,244.853705
+669.494115,75,245.583022
+665.863288,75,246.288591
+675.610515,75,246.997131
+674.364104,75,247.692349
+671.981805,75,248.387023
+671.543337,75,249.238482
+666.691703,75,250.095391
+673.100052,75,250.925414
+668.953962,75,251.650591
+669.503224,75,252.331479
+669.358942,75,253.013347
+668.568008,75,253.68348
+670.225985,75,254.366586
diff --git a/logs/myoHandWheelHoldFixed-v0_5.monitor.csv b/logs/myoHandWheelHoldFixed-v0_5.monitor.csv
index 569533d..d35e996 100644
--- a/logs/myoHandWheelHoldFixed-v0_5.monitor.csv
+++ b/logs/myoHandWheelHoldFixed-v0_5.monitor.csv
@@ -1,30 +1,248 @@
-#{"t_start": 1753224774.0341969, "env_id": "myoHandWheelHoldFixed-v0"}
+#{"t_start": 1753283319.8499439, "env_id": "myoHandWheelHoldFixed-v0"}
 r,l,t
-1013.61444,75,1.343731
-1009.870372,75,1.976817
-1011.110782,75,2.580597
-1009.205111,75,3.199347
-1008.714985,75,3.801077
-995.283846,75,4.427775
-1015.766284,75,5.121675
-1009.971937,75,5.82722
-1013.728506,75,6.459204
-1016.10202,75,7.101366
-1017.371848,75,7.822944
-1013.015825,75,8.46509
-1023.569169,75,9.093041
-1015.788413,75,9.803874
-1016.122623,75,10.66875
-1019.586756,75,11.543143
-1014.751098,75,12.494141
-1021.837124,75,13.3327
-1012.06654,75,14.162922
-1018.630665,75,14.998234
-1016.545357,75,15.812556
-1006.306854,75,16.677098
-1013.127392,75,17.350252
-1003.220007,75,18.043014
-1013.604829,75,18.654527
-1024.187038,75,19.287214
-1015.425385,75,19.914209
-1032.681333,75,36.868812
+652.728712,75,2.804496
+651.084337,75,4.085505
+656.156119,75,5.269161
+657.316309,75,6.527149
+658.0039,75,7.687539
+660.259915,75,8.676836
+659.253509,75,9.72165
+639.153659,75,10.526232
+658.627369,75,11.362557
+661.491333,75,12.306911
+665.256727,75,13.292504
+665.060958,75,14.335572
+659.9223,75,15.326636
+667.546792,75,16.16109
+665.791999,75,16.913166
+636.052024,75,17.637971
+662.433689,75,18.352874
+658.110518,75,19.09647
+654.940988,75,19.809414
+656.844029,75,20.590245
+657.126525,75,21.304302
+658.268621,75,22.028361
+655.806185,75,22.743022
+658.05816,75,23.444129
+650.133248,75,24.138962
+654.140123,75,24.884345
+663.349704,75,25.639062
+644.905686,75,36.654403
+658.752176,75,37.361828
+657.834342,75,38.12873
+659.288742,75,38.857978
+662.005903,75,39.580065
+661.753224,75,40.292187
+656.339636,75,41.016237
+658.130703,75,41.72578
+665.103422,75,42.44046
+668.358886,75,43.137339
+662.08675,75,43.847836
+662.40481,75,44.572594
+668.3837,75,45.279465
+663.961487,75,45.989892
+657.580587,75,46.694117
+661.93909,75,47.409481
+671.165031,75,48.118615
+663.892016,75,48.81939
+660.461451,75,49.566647
+662.106102,75,50.296687
+654.656647,75,51.016424
+655.266213,75,51.73068
+657.13405,75,52.442021
+656.992459,75,53.154029
+664.669074,75,53.872102
+655.131001,75,54.577471
+663.916152,75,55.298072
+662.21706,75,65.285662
+652.991251,75,66.000703
+669.483444,75,66.703553
+652.769601,75,67.418055
+665.207029,75,68.135384
+655.932989,75,68.854019
+657.399062,75,69.555678
+656.44659,75,70.273109
+662.061421,75,70.985643
+660.151078,75,71.679874
+663.785366,75,72.40749
+660.662311,75,73.109225
+664.298045,75,73.814613
+663.492525,75,74.525355
+657.031565,75,75.227983
+662.461069,75,75.943835
+663.513043,75,76.670438
+659.61727,75,77.368267
+656.866857,75,78.063757
+661.975712,75,78.78207
+660.277011,75,79.533494
+660.317564,75,80.26142
+667.44098,75,80.978024
+662.21447,75,81.691947
+657.34672,75,82.412467
+662.06678,75,83.120231
+662.436788,75,83.803339
+655.900376,75,93.441839
+664.822363,75,94.160161
+661.632408,75,94.857225
+665.091184,75,95.551883
+662.06561,75,96.288703
+663.756008,75,96.992906
+652.958798,75,97.685933
+660.099507,75,98.395854
+673.073611,75,99.117337
+619.605803,75,99.820898
+648.103038,75,100.534399
+644.31232,75,101.228333
+656.501286,75,101.925494
+663.599688,75,102.635148
+660.273759,75,103.34528
+662.096101,75,104.074183
+653.847707,75,104.773578
+660.639422,75,105.472124
+662.179517,75,106.188481
+658.94827,75,106.876288
+666.003251,75,107.583286
+661.653215,75,108.297856
+664.057743,75,109.00312
+662.331961,75,109.715884
+655.629424,75,110.443391
+662.256988,75,111.136619
+661.51993,75,111.849884
+664.896914,75,112.552087
+666.207892,75,122.088336
+654.024412,75,122.798478
+668.180393,75,123.496442
+660.117301,75,124.198535
+660.642895,75,124.90105
+648.106401,75,125.618024
+662.325958,75,126.341005
+665.865491,75,127.036912
+660.672281,75,127.743527
+663.022143,75,128.441725
+655.192504,75,129.13902
+665.909313,75,129.848989
+661.350486,75,130.562763
+666.717845,75,131.285102
+654.37612,75,131.998117
+662.612689,75,132.712059
+665.801151,75,133.427374
+650.960317,75,134.155772
+664.569289,75,134.861102
+658.815149,75,135.573803
+657.165413,75,136.303517
+663.420402,75,137.009817
+668.000213,75,137.717664
+666.070902,75,138.436652
+673.580457,75,139.441612
+649.049899,75,140.219918
+655.276153,75,140.914787
+652.327846,75,141.628754
+649.931127,75,150.622093
+663.915418,75,151.293373
+664.560215,75,151.985092
+666.310766,75,152.682545
+664.380036,75,153.363397
+657.813839,75,154.057946
+658.073654,75,154.755696
+662.723509,75,155.442605
+665.79982,75,156.122068
+669.04154,75,156.801263
+668.796558,75,157.49372
+658.536957,75,158.176223
+667.988713,75,158.862005
+658.298276,75,159.540171
+657.513691,75,160.238917
+659.653063,75,160.929759
+663.398375,75,161.612533
+665.826106,75,162.341228
+658.606555,75,163.026387
+667.024628,75,163.702619
+665.549632,75,164.393591
+659.80267,75,165.079143
+670.03216,75,165.778508
+663.416053,75,166.472231
+670.192402,75,167.145324
+668.274492,75,167.820463
+656.261309,75,168.514825
+659.241949,75,177.541316
+661.480969,75,178.235591
+659.870926,75,178.911609
+663.168445,75,179.590486
+667.406355,75,180.278141
+662.942437,75,180.966151
+666.790646,75,181.651074
+667.833637,75,182.359831
+658.944894,75,183.04592
+654.28327,75,183.743253
+663.345287,75,184.431201
+659.34583,75,185.124611
+667.208581,75,185.827123
+662.707052,75,186.520104
+659.301846,75,187.211785
+662.073086,75,187.893078
+671.440289,75,188.573721
+663.514571,75,189.273326
+664.395942,75,189.956994
+656.521599,75,190.658054
+662.161825,75,191.328413
+664.761414,75,192.007376
+673.425943,75,192.684085
+657.654206,75,193.372509
+660.406756,75,194.062832
+666.165561,75,194.747145
+669.725606,75,195.420519
+665.209461,75,205.068457
+668.826394,75,205.768932
+662.558919,75,206.458412
+667.392888,75,207.177519
+669.103967,75,207.893529
+667.23592,75,208.719276
+669.457914,75,210.149144
+671.564355,75,211.019322
+661.030238,75,211.763754
+674.618481,75,212.425357
+656.949043,75,213.124037
+662.447831,75,213.790547
+662.04328,75,214.475045
+673.475824,75,215.177648
+658.600051,75,215.858725
+668.73297,75,216.539716
+667.665549,75,217.233872
+671.560493,75,217.928938
+668.111104,75,218.661324
+664.092978,75,219.412274
+666.003852,75,220.129651
+666.242009,75,220.85429
+665.171351,75,221.574311
+666.422077,75,222.270686
+659.196154,75,222.973734
+667.274395,75,223.706117
+666.384232,75,224.431444
+666.437699,75,225.140489
+668.682769,75,235.121793
+670.669987,75,235.884143
+670.363262,75,236.755145
+671.340531,75,237.574647
+672.078778,75,238.314357
+670.917383,75,239.026623
+674.810349,75,239.737619
+676.807646,75,240.482689
+673.29729,75,241.241331
+670.882978,75,241.96036
+674.321123,75,242.664945
+668.60655,75,243.380005
+670.825802,75,244.110701
+672.880391,75,244.838781
+674.640075,75,245.572127
+668.796127,75,246.275635
+672.474517,75,246.986237
+677.914221,75,247.679428
+670.910012,75,248.378159
+673.414833,75,249.227587
+669.426131,75,250.084496
+672.455522,75,250.914519
+673.11114,75,251.638659
+667.234173,75,252.320584
+671.462731,75,253.002452
+673.247538,75,253.672585
+675.045348,75,254.355691
diff --git a/logs/myoHandWheelHoldFixed-v0_6.monitor.csv b/logs/myoHandWheelHoldFixed-v0_6.monitor.csv
index 09bceb7..224aa30 100644
--- a/logs/myoHandWheelHoldFixed-v0_6.monitor.csv
+++ b/logs/myoHandWheelHoldFixed-v0_6.monitor.csv
@@ -1,30 +1,248 @@
-#{"t_start": 1753224774.0225222, "env_id": "myoHandWheelHoldFixed-v0"}
+#{"t_start": 1753283319.8480902, "env_id": "myoHandWheelHoldFixed-v0"}
 r,l,t
-1009.556028,75,1.355406
-1020.352474,75,1.98749
-1017.684381,75,2.592272
-1016.065644,75,3.210023
-1012.117398,75,3.812752
-1022.143674,75,4.439449
-1012.488937,75,5.130339
-1024.697259,75,5.83589
-1012.505855,75,6.469879
-1019.75989,75,7.11404
-1007.471809,75,7.836618
-1013.16939,75,8.477757
-1017.637341,75,9.104716
-1018.783097,75,9.816557
-1012.359145,75,10.680424
-1022.594206,75,11.553826
-1018.80107,75,12.505816
-1023.706493,75,13.344374
-1014.852135,75,14.176598
-1019.786124,75,15.009909
-1010.253725,75,15.823231
-1025.784169,75,16.685769
-1013.528804,75,17.360925
-1027.666169,75,18.054688
-1024.678987,75,18.664201
-1023.147914,75,19.29989
-1014.01703,75,19.926884
-1032.681333,75,36.882523
+662.598628,75,2.808367
+666.154184,75,4.089116
+663.929658,75,5.273046
+629.165935,75,6.533304
+651.699707,75,7.689392
+656.396797,75,8.680723
+651.698424,75,9.724495
+664.58415,75,10.528086
+659.61313,75,11.372917
+659.845285,75,12.308765
+653.874217,75,13.301358
+662.762517,75,14.336432
+661.83671,75,15.329465
+651.215775,75,16.166956
+656.27668,75,16.916023
+658.816165,75,17.63883
+663.35316,75,18.354727
+658.018081,75,19.098324
+654.642324,75,19.813295
+663.75585,75,20.593111
+654.836709,75,21.309166
+660.011846,75,22.030215
+667.938613,75,22.745876
+653.872667,75,23.447986
+666.374474,75,24.141806
+658.602314,75,24.886198
+666.153671,75,25.640916
+647.200358,75,36.654253
+658.846005,75,37.364683
+662.995072,75,38.132644
+663.392972,75,38.859831
+661.354309,75,39.583917
+660.447889,75,40.294041
+653.696616,75,41.019655
+664.253035,75,41.728632
+662.51291,75,42.443317
+665.769969,75,43.139699
+660.882038,75,43.851689
+653.516584,75,44.577451
+660.041578,75,45.282325
+658.367396,75,45.992746
+651.414878,75,46.697964
+664.055465,75,47.413337
+660.709016,75,48.11847
+653.765774,75,48.824237
+658.580221,75,49.565651
+663.349762,75,50.300532
+665.811379,75,51.019277
+650.983935,75,51.731536
+670.157424,75,52.444871
+658.744196,75,53.155883
+667.551934,75,53.875954
+664.2873,75,54.582341
+657.876453,75,55.299926
+668.499901,75,65.287516
+656.662819,75,66.004572
+652.279016,75,66.707419
+651.725128,75,67.422428
+659.912556,75,68.137238
+656.443848,75,68.857884
+655.175508,75,69.559614
+662.207748,75,70.274963
+659.150705,75,70.987496
+664.506767,75,71.683275
+662.975911,75,72.413872
+657.858302,75,73.113102
+659.430275,75,73.816466
+669.500087,75,74.529225
+660.77028,75,75.229836
+673.699062,75,75.945689
+655.484886,75,76.670277
+653.638129,75,77.370121
+657.896799,75,78.067639
+656.359687,75,78.783923
+654.644715,75,79.535348
+656.719714,75,80.260657
+664.561135,75,80.979877
+658.6115,75,81.695558
+658.033962,75,82.416339
+653.742218,75,83.122084
+649.921712,75,83.807204
+661.103503,75,93.445716
+650.727352,75,94.164027
+651.678471,75,94.861087
+661.11307,75,95.555843
+661.701376,75,96.290557
+660.401092,75,96.99476
+669.271054,75,97.686768
+657.788063,75,98.397708
+658.122455,75,99.119191
+665.271545,75,99.824765
+656.688705,75,100.536252
+661.102762,75,101.232205
+664.929152,75,101.927347
+669.758776,75,102.639018
+656.770608,75,103.34765
+664.053682,75,104.078051
+652.219509,75,104.777444
+675.885908,75,105.475986
+666.510358,75,106.190854
+662.778926,75,106.878142
+654.011863,75,107.587159
+658.081665,75,108.29971
+658.898411,75,109.006984
+657.050244,75,109.71949
+659.85217,75,110.445245
+666.938843,75,111.138473
+663.837506,75,111.851738
+662.005179,75,112.553941
+656.965898,75,122.09019
+658.298472,75,122.800331
+664.286621,75,123.501793
+654.994541,75,124.200389
+665.724013,75,124.902904
+657.87036,75,125.620891
+663.320168,75,126.342859
+648.084247,75,127.038765
+655.036229,75,127.743364
+666.385733,75,128.443578
+664.181439,75,129.142884
+661.697579,75,129.852855
+657.642478,75,130.564616
+650.978293,75,131.286956
+666.420483,75,131.999971
+664.036851,75,132.713913
+658.050933,75,133.431277
+669.821869,75,134.159637
+658.503696,75,134.864969
+659.652724,75,135.575657
+651.755281,75,136.305371
+657.139742,75,137.01571
+660.784877,75,137.72153
+659.315975,75,138.440523
+673.580457,75,139.445493
+609.325383,75,140.223796
+666.605208,75,140.916641
+665.803069,75,141.62859
+652.649277,75,150.623947
+661.087856,75,151.295227
+671.038004,75,151.984936
+663.510898,75,152.684399
+662.155179,75,153.365251
+656.108944,75,154.0598
+649.667126,75,154.759562
+660.668358,75,155.44859
+669.152076,75,156.123921
+667.78502,75,156.801562
+660.596296,75,157.497593
+660.47715,75,158.178076
+670.548193,75,158.865866
+655.711491,75,159.542024
+664.092846,75,160.244295
+662.878561,75,160.934787
+660.248711,75,161.614387
+663.435505,75,162.345095
+661.329708,75,163.026217
+663.806829,75,163.706485
+660.243624,75,164.397461
+665.731262,75,165.078984
+653.628355,75,165.782436
+663.97545,75,166.474085
+658.469922,75,167.149189
+667.309038,75,167.822316
+670.58072,75,168.514664
+662.016404,75,177.543169
+661.025326,75,178.238453
+664.882048,75,178.915479
+666.705912,75,179.59435
+669.748004,75,180.277986
+663.233465,75,180.968004
+668.032525,75,181.652928
+662.334082,75,182.361684
+665.448856,75,183.047774
+666.523783,75,183.745107
+663.466097,75,184.433055
+670.037801,75,185.126465
+665.730245,75,185.828976
+667.224383,75,186.521958
+666.475761,75,187.213639
+663.313734,75,187.894932
+663.739609,75,188.57356
+667.559829,75,189.273166
+670.430017,75,189.958848
+670.142256,75,190.659908
+669.13508,75,191.331778
+659.629562,75,192.009229
+669.159787,75,192.685938
+666.238227,75,193.374362
+669.21301,75,194.064686
+666.964058,75,194.748999
+663.010357,75,195.424387
+661.394168,75,205.07232
+668.14521,75,205.772539
+669.326678,75,206.460265
+670.18553,75,207.179373
+670.460297,75,207.897398
+663.077235,75,208.725153
+669.473564,75,210.154824
+664.888365,75,211.021176
+674.949432,75,211.765608
+668.857627,75,212.429472
+672.0921,75,213.126415
+666.319694,75,213.792401
+666.7806,75,214.476899
+673.149063,75,215.179501
+671.054922,75,215.864237
+666.830866,75,216.541569
+661.286145,75,217.235726
+673.114413,75,217.930792
+673.184604,75,218.664241
+674.438175,75,219.414128
+661.457618,75,220.133543
+665.621588,75,220.858157
+671.572202,75,221.578178
+670.111572,75,222.27254
+666.701396,75,222.974174
+670.472367,75,223.707971
+658.166602,75,224.435321
+666.642649,75,225.140331
+673.71112,75,235.121634
+668.095893,75,235.886508
+674.647287,75,236.759017
+668.842083,75,237.573477
+670.600125,75,238.316211
+668.471108,75,239.028477
+672.73047,75,239.738461
+672.412018,75,240.484543
+666.74207,75,241.243185
+673.503958,75,241.963228
+671.0558,75,242.666799
+666.623568,75,243.381859
+670.550334,75,244.112555
+675.380707,75,244.842649
+674.202766,75,245.573981
+669.388599,75,246.27955
+673.313113,75,246.98809
+672.354432,75,247.681282
+668.682474,75,248.380013
+666.965968,75,249.229441
+669.619751,75,250.08635
+668.630682,75,250.916373
+666.883183,75,251.642075
+674.761257,75,252.322438
+673.25783,75,253.002551
+667.550936,75,253.676452
+673.184486,75,254.357545
diff --git a/logs/myoHandWheelHoldFixed-v0_7.monitor.csv b/logs/myoHandWheelHoldFixed-v0_7.monitor.csv
index 4a78309..2401595 100644
--- a/logs/myoHandWheelHoldFixed-v0_7.monitor.csv
+++ b/logs/myoHandWheelHoldFixed-v0_7.monitor.csv
@@ -1,30 +1,248 @@
-#{"t_start": 1753224773.9938774, "env_id": "myoHandWheelHoldFixed-v0"}
+#{"t_start": 1753283319.8586583, "env_id": "myoHandWheelHoldFixed-v0"}
 r,l,t
-1024.632091,75,1.38605
-1021.527931,75,2.017136
-1017.972814,75,2.619916
-1024.591147,75,3.238668
-1019.046291,75,3.8424
-1022.528901,75,4.466096
-1010.187962,75,5.158984
-1012.296401,75,5.86754
-1014.07443,75,6.499524
-1015.845939,75,7.141685
-1019.568136,75,7.867259
-1012.484834,75,8.505409
-1015.835783,75,9.132359
-1023.017029,75,9.843191
-1011.967579,75,10.711374
-1022.176196,75,11.582471
-1009.925649,75,12.534461
-1007.46589,75,13.373019
-1022.536174,75,14.204239
-1020.324498,75,15.038554
-1003.734591,75,15.851875
-1014.660883,75,16.717417
-1013.068754,75,17.39157
-1013.579878,75,18.084335
-1015.195803,75,18.693845
-1025.608438,75,19.329536
-1020.737955,75,19.954529
-1032.681333,75,36.911168
+660.379008,75,2.797799
+661.128851,75,4.082598
+654.997916,75,5.266544
+670.219254,75,6.51488
+655.430416,75,7.678824
+668.978143,75,8.670155
+654.977923,75,9.712935
+647.399329,75,10.517518
+663.455398,75,11.35587
+668.268188,75,12.298197
+665.836479,75,13.285792
+657.637041,75,14.329854
+660.11601,75,15.318897
+655.196632,75,16.156388
+663.219446,75,16.904452
+656.1421,75,17.629256
+663.495056,75,18.343157
+657.626478,75,19.087755
+659.44316,75,19.802727
+659.936072,75,20.581531
+659.289009,75,21.298598
+659.550497,75,22.019647
+659.329433,75,22.734308
+655.97455,75,23.438434
+660.619626,75,24.131238
+649.329119,75,24.876146
+655.508551,75,25.630348
+658.380444,75,36.644715
+668.957123,75,37.355128
+656.619268,75,38.122076
+662.596668,75,38.849263
+657.40962,75,39.572337
+657.978337,75,40.282476
+662.555976,75,41.007523
+662.006758,75,41.718064
+668.857666,75,42.431746
+660.136687,75,43.12763
+654.878559,75,43.840121
+662.947916,75,44.564883
+663.834178,75,45.270751
+662.529544,75,45.981177
+658.107816,75,46.686408
+661.916302,75,47.400766
+657.022191,75,48.1069
+656.846158,75,48.810675
+661.072772,75,49.557421
+659.435658,75,50.288974
+659.381066,75,51.006703
+660.957351,75,51.721965
+664.833373,75,52.43531
+661.978657,75,53.145315
+660.135375,75,53.865386
+660.168133,75,54.569777
+655.766832,75,55.289358
+660.964411,75,65.276948
+665.659692,75,65.991989
+664.117761,75,66.696851
+664.437308,75,67.41186
+666.561846,75,68.128863
+653.58696,75,68.845304
+651.371246,75,69.548009
+665.248303,75,70.264395
+667.542474,75,70.976928
+657.755301,75,71.672706
+653.899591,75,72.400785
+668.186383,75,73.10051
+659.881204,75,73.805898
+659.79705,75,74.51664
+666.951318,75,75.221786
+661.343016,75,75.935121
+662.978838,75,76.661724
+662.629372,75,77.359552
+660.14623,75,78.055043
+652.825379,75,78.775373
+658.965539,75,79.523268
+663.875217,75,80.252705
+662.874995,75,80.969309
+658.142477,75,81.683233
+660.508928,75,82.405771
+659.163471,75,83.112563
+660.245906,75,83.794624
+656.589249,75,93.433125
+658.492692,75,94.150821
+659.587134,75,94.848511
+664.511535,75,95.543169
+668.836739,75,96.279989
+664.301727,75,96.983176
+659.102881,75,97.676199
+654.546281,75,98.38714
+662.322856,75,99.108623
+655.836926,75,99.814197
+665.433614,75,100.523675
+670.869838,75,101.221637
+652.88555,75,101.918533
+651.355793,75,102.62845
+653.621639,75,103.338127
+654.976692,75,104.065469
+662.476233,75,104.764864
+663.748976,75,105.465417
+653.92045,75,106.180286
+660.497623,75,106.867574
+662.435551,75,107.574571
+654.472087,75,108.291153
+656.691438,75,108.994405
+659.268093,75,109.708921
+668.696744,75,110.432661
+661.638253,75,111.131929
+669.525035,75,111.84117
+650.528452,75,112.54539
+664.02654,75,122.077604
+656.8163,75,122.787755
+654.95307,75,123.489737
+664.52894,75,124.191842
+671.099129,75,124.892335
+659.565097,75,125.60931
+667.035914,75,126.332291
+664.143379,75,127.030219
+666.68125,75,127.734812
+661.403932,75,128.43301
+666.672313,75,129.132315
+652.985695,75,129.840274
+648.887663,75,130.554048
+655.21729,75,131.274372
+655.575641,75,131.989403
+663.537562,75,132.704357
+665.593847,75,133.41866
+660.555703,75,134.147058
+663.566115,75,134.852387
+665.889966,75,135.564495
+665.333017,75,136.292789
+656.101964,75,137.005141
+664.234864,75,137.708949
+662.716915,75,138.427938
+673.580457,75,139.432897
+650.167096,75,140.211204
+660.808924,75,140.908115
+660.083618,75,141.62004
+661.779115,75,150.613379
+665.708247,75,151.284659
+665.721041,75,151.974368
+657.034823,75,152.671821
+662.90746,75,153.356696
+660.656262,75,154.049232
+668.472012,75,154.746981
+660.056206,75,155.435899
+666.851042,75,156.112735
+664.967054,75,156.792548
+665.768812,75,157.485006
+666.981835,75,158.167508
+653.588428,75,158.855298
+659.097997,75,159.530432
+658.477456,75,160.23171
+661.598117,75,160.921045
+661.433103,75,161.604863
+659.869386,75,162.334527
+661.659909,75,163.015649
+663.785223,75,163.695917
+664.953799,75,164.386893
+661.83958,75,165.070429
+647.725689,75,165.771868
+657.159379,75,166.463517
+661.765389,75,167.13661
+663.864017,75,167.811748
+658.629035,75,168.50611
+663.603696,75,177.530593
+663.490605,75,178.227885
+670.445375,75,178.904911
+660.668468,75,179.583782
+663.642443,75,180.269427
+671.667543,75,180.957436
+665.51803,75,181.642359
+663.366941,75,182.349071
+663.292092,75,183.037205
+661.563784,75,183.734539
+664.090695,75,184.421474
+662.930285,75,185.115896
+663.163403,75,185.818408
+668.306618,75,186.513416
+664.799515,75,187.203071
+663.852204,75,187.882354
+659.639989,75,188.565007
+668.31925,75,189.264611
+663.956157,75,189.950292
+664.991241,75,190.646534
+671.194757,75,191.32121
+663.646199,75,191.998661
+662.100593,75,192.673358
+667.368843,75,193.363794
+659.897463,75,194.054117
+664.191052,75,194.738431
+668.956944,75,195.413819
+656.223772,75,205.063767
+663.117296,75,205.760217
+669.489903,75,206.449697
+665.599817,75,207.17082
+664.009864,75,207.888847
+671.267519,75,208.712574
+669.394621,75,210.142462
+662.821063,75,211.010608
+661.977725,75,211.755039
+668.091488,75,212.416643
+662.369408,75,213.115846
+668.47325,75,213.781832
+664.158264,75,214.463288
+663.150283,75,215.168933
+669.191296,75,215.847937
+670.42405,75,216.529625
+667.694208,75,217.225158
+663.907493,75,217.920224
+661.959569,75,218.65261
+666.830919,75,219.40356
+669.133855,75,220.122975
+667.925243,75,220.845576
+671.156099,75,221.56761
+664.196976,75,222.261972
+668.188901,75,222.963605
+665.908104,75,223.697403
+668.074601,75,224.424753
+665.461212,75,225.131775
+671.053267,75,235.113078
+669.33541,75,235.87594
+674.474127,75,236.748449
+666.341537,75,237.562908
+668.351005,75,238.305643
+667.4619,75,239.017909
+671.907435,75,239.728904
+672.855473,75,240.479734
+672.346006,75,241.232617
+668.650066,75,241.951645
+670.834255,75,242.65825
+675.658403,75,243.37129
+671.765522,75,244.101986
+677.19866,75,244.832081
+669.973309,75,245.563413
+671.073347,75,246.268981
+668.374931,75,246.977522
+672.011915,75,247.670714
+669.996576,75,248.367414
+667.600974,75,249.220898
+673.587512,75,250.075782
+668.130575,75,250.909854
+680.217852,75,251.629944
+666.007334,75,252.31187
+671.571703,75,252.993738
+674.982182,75,253.665884
+669.04873,75,254.346977
diff --git a/logs/myoHandWheelHoldFixed-v0_8.monitor.csv b/logs/myoHandWheelHoldFixed-v0_8.monitor.csv
index 2732078..4c5081e 100644
--- a/logs/myoHandWheelHoldFixed-v0_8.monitor.csv
+++ b/logs/myoHandWheelHoldFixed-v0_8.monitor.csv
@@ -1,30 +1,248 @@
-#{"t_start": 1753224774.0128736, "env_id": "myoHandWheelHoldFixed-v0"}
+#{"t_start": 1753283319.839049, "env_id": "myoHandWheelHoldFixed-v0"}
 r,l,t
-1017.388651,75,1.368562
-1022.615702,75,1.997139
-1013.995337,75,2.600919
-1019.86651,75,3.22067
-1012.615317,75,3.8224
-1012.543143,75,4.4471
-1020.051194,75,5.145996
-1013.620724,75,5.846539
-1007.361067,75,6.483528
-1007.690422,75,7.124691
-1001.027247,75,7.847264
-1018.342037,75,8.487406
-1016.554426,75,9.114364
-1017.189355,75,9.825198
-1008.748952,75,10.691065
-1014.108093,75,11.563475
-1013.211152,75,12.516465
-1000.946912,75,13.355024
-1002.067948,75,14.188247
-1005.332073,75,15.017557
-1003.609929,75,15.834881
-1024.330652,75,16.697423
-1017.530785,75,17.371575
-1024.976068,75,18.065338
-1015.872688,75,18.67585
-1011.084531,75,19.31154
-1019.095805,75,19.936533
-1032.681332,75,36.892172
+652.23927,75,2.817408
+660.479178,75,4.093852
+660.997145,75,5.286153
+658.776204,75,6.539501
+664.775991,75,7.700444
+657.715534,75,8.689764
+665.415908,75,9.737534
+660.076238,75,10.537127
+657.491612,75,11.38297
+653.48206,75,12.323802
+665.966956,75,13.3074
+649.283416,75,14.350503
+661.133759,75,15.338506
+661.42314,75,16.176996
+660.206122,75,16.924061
+662.552406,75,17.647871
+665.505411,75,18.366722
+658.189899,75,19.107365
+656.437681,75,19.82732
+661.784134,75,20.60114
+663.120181,75,21.3172
+661.703735,75,22.041273
+666.500608,75,22.753917
+650.257575,75,23.458043
+666.837743,75,24.150847
+655.674511,75,24.896825
+665.40279,75,25.650957
+660.855456,75,36.663294
+659.632697,75,37.373724
+653.624154,75,38.14065
+642.248829,75,38.869879
+660.421589,75,39.591946
+658.124193,75,40.303082
+651.30741,75,41.027132
+661.725922,75,41.736674
+667.035877,75,42.451355
+666.70771,75,43.148741
+655.858323,75,43.858731
+647.624565,75,44.590779
+657.865818,75,45.292873
+657.008519,75,46.000786
+656.586473,75,46.706017
+643.711092,75,47.422378
+660.655842,75,48.130511
+652.729963,75,48.83128
+665.288573,75,49.577542
+650.588783,75,50.308583
+663.466816,75,51.026312
+657.071639,75,51.740577
+657.536712,75,52.454919
+661.045711,75,53.164924
+658.932432,75,53.882997
+663.94288,75,54.593394
+659.122939,75,55.309996
+664.903762,75,65.296557
+656.695158,75,66.011598
+658.808877,75,66.71646
+660.549505,75,67.42895
+662.106962,75,68.148472
+665.646744,75,68.864913
+661.552573,75,69.568655
+668.785695,75,70.284004
+669.276004,75,70.996537
+659.256325,75,71.692316
+660.539537,75,72.420395
+655.691662,75,73.120119
+661.595362,75,73.825507
+668.199383,75,74.53625
+661.896562,75,75.238877
+663.294132,75,75.95473
+665.735487,75,76.679318
+646.607841,75,77.381182
+662.228195,75,78.074652
+666.756064,75,78.800418
+661.884329,75,79.546407
+667.425318,75,80.270746
+663.579342,75,80.988918
+663.647874,75,81.704599
+660.369034,75,82.42538
+656.738208,75,83.130086
+657.842078,75,83.814233
+656.905924,75,93.452734
+657.938114,75,94.171056
+655.601972,75,94.86812
+651.502552,75,95.563826
+655.672777,75,96.299598
+669.241663,75,97.003801
+660.502897,75,97.693798
+646.984698,75,98.406749
+661.817313,75,99.13025
+658.444461,75,99.833806
+664.921136,75,100.543284
+648.44651,75,101.241246
+658.154476,75,101.938143
+668.00063,75,102.646043
+667.051362,75,103.35721
+650.5748,75,104.089108
+660.229942,75,104.784473
+663.757332,75,105.485027
+662.620581,75,106.199895
+657.309637,75,106.887183
+665.986015,75,107.5962
+658.568338,75,108.310762
+659.161867,75,109.016025
+659.877232,75,109.732566
+662.151146,75,110.454286
+659.958377,75,111.149527
+653.250308,75,111.862789
+656.029028,75,112.562982
+663.440733,75,122.099231
+654.742714,75,122.809372
+657.646395,75,123.510834
+661.494514,75,124.215521
+644.288318,75,124.910938
+662.41159,75,125.629932
+660.341747,75,126.3519
+657.065528,75,127.049829
+668.558679,75,127.752405
+649.438155,75,128.452619
+656.930792,75,129.149915
+664.981808,75,129.859883
+657.129354,75,130.573658
+663.286888,75,131.293981
+660.06079,75,132.012445
+657.658183,75,132.723966
+669.06633,75,133.438269
+663.773161,75,134.166667
+654.853412,75,134.871997
+654.535144,75,135.584698
+660.921816,75,136.314412
+652.405018,75,137.022724
+661.576785,75,137.730571
+655.342955,75,138.447547
+673.593895,75,139.45659
+653.781007,75,140.232837
+664.490911,75,140.925682
+659.66118,75,141.635877
+649.517801,75,150.63474
+654.344128,75,151.304268
+664.162184,75,152.001662
+663.180853,75,152.69344
+664.779232,75,153.374292
+667.734249,75,154.068841
+662.410174,75,154.766591
+664.588215,75,155.457095
+660.147762,75,156.133496
+668.483887,75,156.81164
+668.828296,75,157.506634
+662.649091,75,158.187117
+659.340019,75,158.874907
+670.500662,75,159.551065
+669.386324,75,160.251319
+653.715552,75,160.943828
+657.588266,75,161.623428
+662.017603,75,162.354136
+660.413848,75,163.035258
+666.163753,75,163.715526
+665.657686,75,164.404486
+668.581007,75,165.090038
+657.396618,75,165.791477
+655.067848,75,166.483126
+664.078208,75,167.15823
+661.842718,75,167.831357
+658.36772,75,168.523705
+660.958291,75,177.550202
+666.993724,75,178.247494
+660.576394,75,178.92452
+672.773739,75,179.603391
+664.123051,75,180.289036
+664.521422,75,180.979055
+666.704539,75,181.66398
+663.1668,75,182.370725
+663.756235,75,183.056815
+665.829257,75,183.754148
+664.192136,75,184.441083
+663.874151,75,185.135506
+668.057264,75,185.836001
+660.318969,75,186.530999
+670.493943,75,187.22268
+660.189795,75,187.903973
+660.416773,75,188.584616
+663.939067,75,189.28422
+654.9919,75,189.969901
+661.739341,75,190.666143
+665.625833,75,191.340819
+663.089697,75,192.01827
+665.206034,75,192.692968
+664.209779,75,193.385414
+663.254517,75,194.073727
+670.378305,75,194.760552
+663.270548,75,195.433428
+666.39858,75,205.085387
+654.366483,75,205.779827
+666.600873,75,206.469306
+664.598685,75,207.190429
+657.57054,75,207.906439
+667.001662,75,208.734194
+669.60912,75,210.162072
+663.126486,75,211.032236
+670.940375,75,211.774649
+674.831445,75,212.436252
+664.783069,75,213.135456
+667.483619,75,213.801442
+665.312093,75,214.482897
+661.442594,75,215.186928
+665.132171,75,215.871174
+667.483911,75,216.549234
+662.170533,75,217.243259
+665.925422,75,217.939833
+670.890575,75,218.677006
+667.894262,75,219.421155
+662.387769,75,220.142584
+665.911711,75,220.867198
+672.9498,75,221.585205
+667.532257,75,222.281581
+666.187719,75,222.983215
+660.691551,75,223.717012
+660.224326,75,224.442338
+659.447627,75,225.151384
+664.62465,75,235.132687
+669.449358,75,235.895549
+669.548127,75,236.768058
+665.903449,75,237.585542
+667.741335,75,238.325252
+676.65653,75,239.037518
+672.870698,75,239.745488
+670.730728,75,240.493584
+670.81424,75,241.252226
+673.030837,75,241.972269
+664.768817,75,242.67786
+669.354175,75,243.3909
+669.058708,75,244.121596
+668.37025,75,244.85169
+670.975477,75,245.583022
+674.619314,75,246.288591
+675.329275,75,246.997131
+673.413583,75,247.690323
+673.089941,75,248.387023
+672.909949,75,249.238482
+665.829674,75,250.095391
+670.26103,75,250.929463
+670.987273,75,251.65007
+675.015444,75,252.331479
+673.538435,75,253.013347
+674.29063,75,253.68348
+671.05063,75,254.366586
diff --git a/myosuite/envs/myo/myowheelchair/wheelhold_v0_push.py b/myosuite/envs/myo/myowheelchair/wheelhold_v0_push.py
deleted file mode 100644
index 735fbd6..0000000
--- a/myosuite/envs/myo/myowheelchair/wheelhold_v0_push.py
+++ /dev/null
@@ -1,194 +0,0 @@
-""" =================================================
-# Copyright (c) Facebook, Inc. and its affiliates
-Authors  :: Vikash Kumar (vikashplus@gmail.com), Vittorio Caggiano (caggiano@gmail.com)
-================================================= """
-
-## PUSHING WHEELCHAIR ##
-
-# TODO traing from 174109, make fin open 0, make wheel rotation 15???
-
-import collections
-import numpy as np
-import math
-from myosuite.utils import gym
-
-from myosuite.envs.myo.base_v0 import BaseV0
-
-
-class WheelHoldFixedEnvV0(BaseV0):
-
-    DEFAULT_OBS_KEYS = ['time', 'wheel_err_right', 'wheel_angle', 'hand_qpos', 'hand_qvel']
-    DEFAULT_RWD_KEYS_AND_WEIGHTS = {
-        "goal_dist": 10.0,
-        "hand_dist" : 0.0,
-        "fin_open": -15.0,
-        "bonus": 0.0,
-        "penalty": 2.0,
-        "wheel_rotation": 15.0,
-        "rotation_bonus": 2.0
-    }
-
-    def __init__(self, model_path, obsd_model_path=None, seed=None, **kwargs):
-
-        # EzPickle.__init__(**locals()) is capturing the input dictionary of the init method of this class.
-        # In order to successfully capture all arguments we need to call gym.utils.EzPickle.__init__(**locals())
-        # at the leaf level, when we do inheritance like we do here.
-        # kwargs is needed at the top level to account for injection of __class__ keyword.
-        # Also see: https://github.com/openai/gym/pull/1497
-        gym.utils.EzPickle.__init__(self, model_path, obsd_model_path, seed, **kwargs)
-
-        # This two step construction is required for pickling to work correctly. All arguments to all __init__
-        # calls must be pickle friendly. Things like sim / sim_obsd are NOT pickle friendly. Therefore we
-        # first construct the inheritance chain, which is just __init__ calls all the way down, with env_base
-        # creating the sim / sim_obsd instances. Next we run through "setup"  which relies on sim / sim_obsd
-        # created in __init__ to complete the setup.
-        super().__init__(model_path=model_path, obsd_model_path=obsd_model_path, seed=seed, env_credits=self.MYO_CREDIT)
-
-        self._setup(**kwargs)
-
-
-    def _setup(self,
-            obs_keys:list = DEFAULT_OBS_KEYS,
-            weighted_reward_keys:list = DEFAULT_RWD_KEYS_AND_WEIGHTS,
-            **kwargs,
-        ):
-        self.goal_sid_right = self.sim.model.site_name2id("wheelchair_grip_right")
-        self.palm_r = self.sim.model.site_name2id("palm_r")
-        self.hand_start_right = self.sim.model.site_name2id("hand_start_right")
-        self.rail_bottom_right = self.sim.model.site_name2id("rail_bottom_right")
-
-        # define the palm and tip site id.
-        # self.palm_r = self.sim.model.site_name2id('S_grasp')
-        self.init_palm_z = self.sim.data.site_xpos[self.palm_r][-1]
-        self.fin0 = self.sim.model.site_name2id("THtip")
-        self.fin1 = self.sim.model.site_name2id("IFtip")
-        self.fin2 = self.sim.model.site_name2id("MFtip")
-        self.fin3 = self.sim.model.site_name2id("RFtip")
-        self.fin4 = self.sim.model.site_name2id("LFtip")
-
-        self.wheel_joint_id = self.sim.model.joint_name2id("right_rear")
-        self.init_wheel_angle = self.sim.data.qpos[self.wheel_joint_id]
-
-
-        #self.goal_sid_left = self.sim.model.site_name2id("wheel_grip_goal_left")
-        #self.object_init_pos = self.sim.data.site_xpos[self.object_sid].copy()
-
-        super()._setup(obs_keys=obs_keys,
-                    weighted_reward_keys=weighted_reward_keys,
-                    **kwargs,
-        )
-        
-        self.init_qpos = self.sim.model.key_qpos[0].copy() # copy the sitting + grabbing wheels keyframe
-
-
-    def get_obs_vec(self):
-        self.obs_dict['time'] = np.array([self.sim.data.time])
-        self.obs_dict['hand_qpos'] = self.sim.data.qpos[13:].copy()
-        self.obs_dict['hand_qvel'] = self.sim.data.qvel[12:].copy()*self.dt
-        #self.obs_dict['wheel_pos'] = self.sim.data.site_xpos[self.object_sid]
-        self.obs_dict['wheel_err_right'] = self.sim.data.site_xpos[self.goal_sid_right] - self.sim.data.site_xpos[self.palm_r]
-        self.obs_dict['hand_initpos_err_right'] = self.sim.data.site_xpos[self.hand_start_right]- self.sim.data.site_xpos[self.goal_sid_right]
-
-        self.obs_dict["palm_pos"] = self.sim.data.site_xpos[self.palm_r]
-        self.obs_dict['fin0'] = self.sim.data.site_xpos[self.fin0]
-        self.obs_dict['fin1'] = self.sim.data.site_xpos[self.fin1]
-        self.obs_dict['fin2'] = self.sim.data.site_xpos[self.fin2]
-        self.obs_dict['fin3'] = self.sim.data.site_xpos[self.fin3]
-        self.obs_dict['fin4'] = self.sim.data.site_xpos[self.fin4]
-
-        self.obs_dict["rail_bottom_right"] = self.sim.data.site_xpos[self.rail_bottom_right]
-
-        self.obs_dict['wheel_angle'] = np.array([self.sim.data.qpos[self.wheel_joint_id]])
-
-
-        if self.sim.model.na>0:
-            self.obs_dict['act'] = self.sim.data.act[:].copy()
-
-        t, obs = self.obsdict2obsvec(self.obs_dict, self.obs_keys)
-        return obs
-
-    def get_obs_dict(self, sim):
-        obs_dict = {}
-        obs_dict['time'] = np.array([sim.data.time])
-        obs_dict['hand_qpos'] = sim.data.qpos[13:].copy()
-        obs_dict['hand_qvel'] = sim.data.qvel[12:].copy()*self.dt
-        #obs_dict['wheel_pos'] = sim.data.site_xpos[self.object_sid]
-        #obs_dict['wheelchair_grip_right'] = sim.data.site_xpos[self.goal_sid] - sim.data.site_xpos[self.object_sid]
-        obs_dict['wheel_err_right'] = sim.data.site_xpos[self.goal_sid_right] - sim.data.site_xpos[self.palm_r]
-        obs_dict['hand_initpos_err_right'] = sim.data.site_xpos[self.hand_start_right]- sim.data.site_xpos[self.goal_sid_right]
-        #add the initial and end target points
-        #could add the fingertips here,
-        obs_dict["palm_pos"] = sim.data.site_xpos[self.palm_r]
-        obs_dict['fin0'] = sim.data.site_xpos[self.fin0]
-        obs_dict['fin1'] = sim.data.site_xpos[self.fin1]
-        obs_dict['fin2'] = sim.data.site_xpos[self.fin2]
-        obs_dict['fin3'] = sim.data.site_xpos[self.fin3]
-        obs_dict['fin4'] = sim.data.site_xpos[self.fin4]
-
-        obs_dict["rail_bottom_right"] = sim.data.site_xpos[self.rail_bottom_right]
-
-        obs_dict['wheel_angle'] = np.array([sim.data.qpos[self.wheel_joint_id]])
-
-
-        #obs_dict['wheel_err_left'] = sim.data.site_xpos[self.goal_sid] - sim.data.site_xpos[self.object_sid]
-        if sim.model.na>0:
-            obs_dict['act'] = sim.data.act[:].copy()
-        
-        return obs_dict
-
-    def get_reward_dict(self, obs_dict):
-        dist_right = np.linalg.norm(obs_dict['wheel_err_right'])
-        hand_initpos_err_right = np.linalg.norm(obs_dict['hand_initpos_err_right'])
-
-        #for wheel rotation
-        wheel_angle_now = self.sim.data.qpos[self.wheel_joint_id]
-        wheel_rotation = wheel_angle_now - self.init_wheel_angle
-        wheel_target = np.pi / 2  # 90 degrees
-
-        wheel_rotation_err = abs(wheel_rotation - wheel_target)
-        wheel_rotation_rwd = math.exp(-5.0 * wheel_rotation_err)
-
-        
-        
-        act_mag = np.linalg.norm(self.obs_dict['act'], axis=-1)/self.sim.model.na if self.sim.model.na !=0 else 0
-        drop = dist_right > 0.500
-
-        fin_keys = ['fin0', 'fin1', 'fin2', 'fin3', 'fin4']
-        # for fin in fin_keys:
-        #     print(fin, type(obs_dict[fin]), np.shape(obs_dict[fin]))
-        fin_open = sum(
-            np.linalg.norm(obs_dict[fin].squeeze() - obs_dict['rail_bottom_right'].squeeze(), axis=-1)
-            for fin in fin_keys
-        )
-        
-        # grip_right = self._check_hand_grip_contact(
-        #     hand_geom_names=["right_index_tip", "right_thumb_tip"],
-        #     wheel_geom_names=[f"handrail_coll{i}" for i in range(1, 17)]
-        # )
-
-        rwd_dict = collections.OrderedDict((
-            ('goal_dist', math.exp(-2.0*abs(dist_right))), #exp(- k * abs(x))
-            ('hand_dist', math.exp(-1.0*abs(hand_initpos_err_right))),
-            ('bonus', 1.*(dist_right<2*0) + 1.*(dist_right<0)),
-            ('act_reg', -1.*act_mag),
-            ("fin_open", np.exp(-20 * fin_open)),  # fin_open + np.log(fin_open +1e-8)
-
-            #('grip_bonus', 1.0 * grip_right),
-            ('penalty', -1.*drop),
-            ('sparse', dist_right < 0.055),
-            #('sparse', 1.0 * grip_right - dist_right),
-            ('solved', dist_right < 0.001 and wheel_rotation_err < 0.05),
-            #('solved', grip_right and dist_right < 0.015),
-            ('done', dist_right > 0.9),
-            ('wheel_rotation', wheel_rotation_rwd),
-            ('rotation_bonus', 1.0 if wheel_rotation_err < 0.05 else 0.0),
-        ))
-        
-        rwd_dict['dense'] = np.sum([wt*rwd_dict[key] for key, wt in self.rwd_keys_wt.items()], axis=0)
-        
-        return rwd_dict
-    
-    def reset(self, **kwargs):
-        self.robot.sync_sims(self.sim, self.sim_obsd)
-        obs = super().reset(**kwargs)
-        return obs
